packages/babel-generator/src/node/parentheses.js

```diff
@@ -32,6 +32,15 @@ const isClassExtendsClause = (node: Object, parent: Object): boolean =>
   (t.isClassDeclaration(parent) || t.isClassExpression(parent)) &&
   parent.superClass === node;
 
+const isPostfixExpression = (node: Object, parent: Object) =>
```

Comment:
AFAIK the postfix expression is used to describe a way of how the operator and its operands are arranged, so I am confused about the term here.

The Spec provides a term [`LeftHandSideExpression` ](https://tc39.es/ecma262/#prod-LeftHandSideExpression) and I think we can stick with that.

Reply:
`LeftHandSideExpression` doesn't work, because it also covers `PrimaryExpression`.

I initially called this "postfix" because TS's `!` is a postfix operator, and because in `foo()` and `foo.bar`, `()` and `.bar` are "postfixes" of the main `foo` expression :sweat_smile:

Maybe `hasPostfixPart`?

Reply:
I'm fine with `hasPostfixPart`.

Reply:
Good for me 🌝